README.md

```diff
@@ -1,3 +1,9 @@
 # Apache APISIX Katacoda scenarios
 
-Apache APISIX scenarios on Katacoda
+See the Katacoda Scenarios at https://www.katacoda.com/api7ai
+
+# Scenarios
+
+1. APISIX Example Scenario 
```

Comment:
Please use Apache APISIX

Reply:
Ohh Okay 😅 , since this is just a test, I did not put much thought into it. It will be changed by the time we will have the first tutorial. I am changing it.